<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundImageConf</class>
 <widget class="QWidget" name="BackgroundImageConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackgroundImageConf</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="backgroundImageSettingsArea" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="backgroundImageLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Background Path : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="backgroundImagePath">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="backgroundImagePreviewArea" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="backgroundImagePreview" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="backgroundImagePreviewLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="imagewidthHeightArea" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelImageWidth">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Width :  0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelImageHeight">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Height :  0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
